src/main/java/Deadline.java

```diff
@@ -0,0 +1,14 @@
+public class Deadline extends Task {
```

Comment:
I like how the indentation and placement of braces in this class follow good programming practices, making the code easier to read and understand.  The code is well-structured and easy to read. The class name, variables and methods are named clearly and appropriately. :wink: